### PR TITLE
Build transferred from Ant to Gradle (Issue #6 resolved)

### PR DESCRIPTION
**Pull request by Miller189**

### CODE REVIEW

**Comment by Miller189** [comment]
"gradlew clean build", "gradlew run" works, no matter what Travis says

**Comment by Miller189** [comment]
The members of this team were myself, Matt Barrett, Randy Forte, and Brandon Hubbard.  Let me know if you need a more specific postmortem.

**Comment by rsanchez-wsu** [comment]
@Miller189 Whoa!  This is quite impressive.  I will need some time to digest this, but I will do my best to get it merged today.  Excellent work from everyone on the team.

If each participant could provide a few sentences to a paragraph explaining his or her particular role in the effort, that would help with the grading.  Each person can submit their own individually here to the pull request, or you can collect them and submit them in a single comment:

CC: @barrett-matthew @Bhulbert2096 @RandyForte

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ language: java
 before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
+  - "chmod +x gradlew"
```
Comment (rsanchez-wsu) [inline .travis.yml]: This should not be necessary.  If you execute `chmod +x gradlew` on your local machine then commit it, Git will remember the permissions.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+apply plugin: 'application'
+
+/**
+* This line specifies where the main.java file is found.
+*/
+
+mainClassName='View.Main'
+
+/**
+* This statement declares where Gradle is to find directories
```
Comment (rsanchez-wsu) [inline build.gradle]: This trailing white space needs to be removed.

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+sourceSets {
+    main {
+		java {
+			srcDirs = ['src']
+		}
+		resources {
+			srcDirs = ['src']
+		}
+	}
+
```
Comment (rsanchez-wsu) [inline build.gradle]: This trailing white space needs to be removed.

File: build.gradle [hunk 4]
```diff
@@ -0,0 +1,70 @@
+/**
```
Comment (rsanchez-wsu) [inline build.gradle]: Please add a license/copyright header (you can copy from one of the Java files) with attribution to those who were involved in developing this file.

File: .travis.yml (hunk 1 above)
Comment (Miller189) [reply .travis.yml]: Finished

File: build.gradle (hunk 4 above)
Comment (Miller189) [reply build.gradle]: Finished

File: build.gradle (hunk 2 above)
Comment (Miller189) [reply build.gradle]: Deleted

File: build.gradle (hunk 3 above)
Comment (Miller189) [reply build.gradle]: Deleted

**Comment by Miller189** [comment]
As far as who did what (chronologically):
  I made the initial gradle build (the basic one with just “import build.xml”) and the first couple of dependencies, Brandon, Matt, & Randy all worked in tandem to resolve the remaining dependencies, Matt powered through the final sourceSet declarations after we spoke with you on Tuesday, Randy optimized the build.gradle file for efficiency and style, and finally I added comments to build.gradle.

**Comment by rsanchez-wsu** [comment]
@Miller189 Note that I also made a top-level comment on the pull request asking that you remove the reports directory and then add a .gitignore entry to ensure that it doesn't get added again by mistake.

File: build.gradle [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+* The application plugin facilitates creating an executable JVM application.
```
Comment (rsanchez-wsu) [inline build.gradle]: Please indent the subsequent lines by one space to follow the convention for multi-line comments.  The format of the license header is correct, for example.

File: build.gradle [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+*/
+
+apply plugin: 'application'
+
+/**
+* This line specifies where the main.java file is found.
```
Comment (rsanchez-wsu) [inline build.gradle]: Please indent the subsequent lines by one space to follow the convention for multi-line comments.  The format of the license header is correct, for example.

File: build.gradle [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+*/
+
+mainClassName='View.Main'
+
+/**
+* This statement declares where Gradle is to find directories
```
Comment (rsanchez-wsu) [inline build.gradle]: Please indent the subsequent lines by one space to follow the convention for multi-line comments.  The format of the license header is correct, for example.

File: build.gradle [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+* for source code, external resources, and test classes
+*/
+
+sourceSets {
+    main {
```
Comment (rsanchez-wsu) [inline build.gradle]: The indentation here should be a tab instead of multiple spaces.

File: build.gradle [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+	test {
+		java {
+			srcDirs = ["Test"]
+		}
+	}
+}
+
+/**
+* Gradle uses this built-in function to locate declared
```
Comment (rsanchez-wsu) [inline build.gradle]: Please indent the subsequent lines by one space to follow the convention for multi-line comments.  The format of the license header is correct, for example.

File: build.gradle [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+* dependencies within Maven's central repositories.
+*/
+
+repositories {
+	mavenCentral()
+}
+
+/**
+* Gradle grabs the JUnit dependencies from Maven central,
```
Comment (rsanchez-wsu) [inline build.gradle]: Please indent the subsequent lines by one space to follow the convention for multi-line comments.  The format of the license header is correct, for example.

File: build.gradle [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+*/
+
+dependencies {
+	testCompile 'junit:junit:4.12'
+	compile fileTree(dir: 'lib')
+	compile fileTree(dir: 'src')
+}
+
+/**
+* IgnoreFailures is set to true until the failure issues are resolved
```
Comment (rsanchez-wsu) [inline build.gradle]: Please indent the subsequent lines by one space to follow the convention for multi-line comments.  The format of the license header is correct, for example.

File: build.gradle [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+* (refer to github.com/rsanchez-wsu/RaiderPlanner/issues/21)
+*/
+
+test {
+	ignoreFailures = true
+}
+
+/**
+* Tells Gradle to assemble a JAR archive.
```
Comment (rsanchez-wsu) [inline build.gradle]: Please indent the subsequent lines by one space to follow the convention for multi-line comments.  The format of the license header is correct, for example.

**Comment by rsanchez-wsu** [comment]
@Miller189 I saw that the .gitignore entry is there, so now the directory and files need to just be removed.

File: build.gradle (hunk 5 above)
Comment (Miller189) [reply build.gradle]: finished & pushed

File: build.gradle (hunk 6 above)
Comment (Miller189) [reply build.gradle]: finished & pushed

File: build.gradle (hunk 7 above)
Comment (Miller189) [reply build.gradle]: finished & pushed

File: build.gradle (hunk 8 above)
Comment (Miller189) [reply build.gradle]: finished & pushed

File: build.gradle (hunk 9 above)
Comment (Miller189) [reply build.gradle]: finished & pushed

File: build.gradle (hunk 10 above)
Comment (Miller189) [reply build.gradle]: finished & pushed

File: build.gradle (hunk 11 above)
Comment (Miller189) [reply build.gradle]: finished & pushed

File: build.gradle (hunk 12 above)
Comment (Miller189) [reply build.gradle]: finished & pushed

File: build.gradle [hunk 13]
```diff
@@ -18,32 +18,23 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
-
-/**
-* The application plugin facilitates creating an executable JVM application.
-*/
+
```
Comment (rsanchez-wsu) [inline build.gradle]: This trailing white space needs to be removed.

File: build.gradle [hunk 14]
```diff
@@ -18,32 +18,23 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
-
-/**
-* The application plugin facilitates creating an executable JVM application.
-*/
+ 
+ /**
```
Comment (rsanchez-wsu) [inline build.gradle]: This slash in the first line of a multi-line comment does not get indented with a space.

File: build.gradle [hunk 15]
```diff
@@ -18,32 +18,23 @@
...
-*/
 
-eclipse.project {
-  	natures 'org.springsource.ide.eclipse.gradle.core.nature'
-}
-
-/**
-* This line specifies where the main.java file is found.
-*/
+ /**
```
Comment (rsanchez-wsu) [inline build.gradle]: This slash in the first line of a multi-line comment does not get indented with a space.

File: build.gradle [hunk 16]
```diff
@@ -18,32 +18,23 @@
...
+ * This line specifies where the main.java file is found.
+ */
 
 mainClassName='View.Main'
 
-/**
-* This statement declares where Gradle is to find directories
-* for source code, external resources, and test classes
-*/
+ /**
```
Comment (rsanchez-wsu) [inline build.gradle]: This slash in the first line of a multi-line comment does not get indented with a space.

File: build.gradle [hunk 17]
```diff
@@ -62,38 +53,38 @@ sourceSets {
 	}
 }
 
-/**
-* Gradle uses this built-in function to locate declared
-* dependencies within Maven's central repositories.
-*/
+ /**
```
Comment (rsanchez-wsu) [inline build.gradle]: This slash in the first line of a multi-line comment does not get indented with a space.

File: build.gradle [hunk 18]
```diff
@@ -62,38 +53,38 @@ sourceSets {
...
 
 repositories {
 	mavenCentral()
 }
 
-/**
-* Gradle grabs the JUnit dependencies from Maven central,
-* and grabs local dependencies from the project filetree.
-*/
+ /**
```
Comment (rsanchez-wsu) [inline build.gradle]: This slash in the first line of a multi-line comment does not get indented with a space.

File: build.gradle [hunk 19]
```diff
@@ -62,38 +53,38 @@ sourceSets {
...
 	testCompile 'junit:junit:4.12'
 	compile fileTree(dir: 'lib')
 	compile fileTree(dir: 'src')
 }
 
-/**
-* IgnoreFailures is set to true until the failure issues are resolved
-* (refer to github.com/rsanchez-wsu/RaiderPlanner/issues/21)
-*/
+ /**
```
Comment (rsanchez-wsu) [inline build.gradle]: This slash in the first line of a multi-line comment does not get indented with a space.

File: build.gradle [hunk 20]
```diff
@@ -62,38 +53,38 @@ sourceSets {
...
+ */
 
 test {
 	ignoreFailures = true
 }
 
-/**
-* Tells Gradle to assemble a JAR archive.
-*/
+ /**
```
Comment (rsanchez-wsu) [inline build.gradle]: This slash in the first line of a multi-line comment does not get indented with a space.

File: build.gradle (hunk 13 above)
Comment (Miller189) [reply build.gradle]: Finished & pushed

File: build.gradle (hunk 14 above)
Comment (Miller189) [reply build.gradle]: Finished & pushed

File: build.gradle (hunk 15 above)
Comment (Miller189) [reply build.gradle]: Finished & pushed

File: build.gradle (hunk 16 above)
Comment (Miller189) [reply build.gradle]: Finished & pushed

File: build.gradle (hunk 17 above)
Comment (Miller189) [reply build.gradle]: Finished & pushed

File: build.gradle (hunk 18 above)
Comment (Miller189) [reply build.gradle]: Finished & pushed

File: build.gradle (hunk 19 above)
Comment (Miller189) [reply build.gradle]: Finished & pushed

File: build.gradle (hunk 20 above)
Comment (Miller189) [reply build.gradle]: Finished & pushed

**Comment by Miller189** [comment]
I have added the Gradle project nature and pushed the changed .project and .classpath files to my branch

**Comment by Miller189** [comment]
Matt modified the permissions so I believe that brings us up to spec

**Pull request merged**